requirements.txt

```diff
@@ -1,7 +1,7 @@
 # Installed by "pip install -e .[dev]" as well as "conda env create --file environment.yml"
 lxml           # parser used for ExoMol website
 hjson          # Json with comments (for default_radis.json)
-publib>=0.4.0  # Plotting styles for Matplotlib. Version, see #647
+publib>=0.3.2  # Plotting styles for Matplotlib. Version, update to 0.4.0 needed for matplotlib==3.8. However, only 0.3.2 is compatible with python==3.8 (see #647)
```

Comment:
therefore test on python>=3.11 or something?

Reply:
sorry I didn't get it.

Reply:
vaex is not compatible with python>3.8 (see original post). matplotlib==3.8 requires python>3.8, thus, we should hold this update for the moment

Reply:
note: vaex is compatible up to python 3.10